draft-fossati-tls-attestation.md

```diff
@@ -218,18 +219,18 @@ of operation when used with TLS, namely:
 - TLS server is the attester, and
 - TLS client and server mutually attest towards each other. 
 
-We will show the message exchanges of the three cases in
-sub-sections below.
+We will show the message exchanges of the first two cases in sub-sections
+below. The last case will be described in future drafts.
```

Comment:
We can be more precise:
```suggestion
We will show the message exchanges of the first two cases in {{attester-cli}} and {{attester-srv}}.
The last case is obtained by composing the first two.
```

This needs adding anchors to the referenced sections like this:
```
## TLS Client as Attester {#attester-cli}
```
and
```
## TLS Server as Attester {#attester-srv}
```